frameworks/helloworld/src/main/dist/svc.yml

```diff
@@ -21,7 +21,6 @@ pods:
           type: ROOT
           size: {{HELLO_DISK}}
         env:
-          PARSE_EMPTY_KEY_AS_BLANK_TEST:
```

Comment:
This was intentionally introduced iirc. it was added as a unit test for a ui parsing bug fix. any reason to remove this?

Reply:
It prevents the quota upgrade/downgrade tests as the service-spec diverges for the hello pod (which doesn't have this ENV var). I'd rather introduce this as a seperate scenario we can test for.